webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PolicyWriterService.scala

```diff
@@ -855,28 +856,41 @@ class PolicyWriterServiceImpl(
       UIO.unit
     } else {
       for {
-        mvOptions  <- getMoveOptions(sortedFolder.head)
```

Comment:
I don't think we need this now

Reply:
Hum, you're right, we need to keep the atomic move if we are on the same FS. In all case, it will give information related to performance (the warning message).